src/weatherAnalysis.java

```diff
@@ -0,0 +1,48 @@
+import java.util.*;
+
+public class weatherAnalysis extends Averager{
+    public static void main(String[] args) {
+        int[] tempArray = intergerify(args);
+        analyzer(tempArray);
+    }
+
+    public static String analyzer(int[] arg){
+        HashSet<Integer> temperatures = new HashSet<>();
+        HashSet<Integer> allTemps = new HashSet<>();
+        int max = -500;
+        int min = 500;
+        for(int currentVal : arg){
+            if(currentVal>max){
+                max = currentVal;
+            }
+            if(currentVal<min){
+                min = currentVal;
+            }
+            temperatures.add(currentVal);
+        }
```

Comment:
For this next section, you can be more concise:
```
System.out.println("Max: " + max);
System.out.println("Min: " + min);
for (int currentTemp = min; currentTemp < max; currentTemp++) {
    allTemps.add(currentTemp);
    if (!temperatures.contains(currentTemp)) {
        System.out.println("Never saw: " + currentTemp);
    }
}
```
This eliminates the need to loop through `alltemps` again